pyproject.toml

```diff
@@ -15,27 +15,26 @@ classifiers=[
     'Operating System :: OS Independent',
     'Programming Language :: Python',
     'Programming Language :: Python :: 3',
-    'Programming Language :: Python :: 3.7',
     'Programming Language :: Python :: 3.8',
     'Programming Language :: Python :: 3.9',
     'Programming Language :: Python :: 3.10',
     'Programming Language :: Python :: 3.11',
+    'Programming Language :: Python :: 3.12',
     'Framework :: Django',
-    'Framework :: Django :: 3',
     'Framework :: Django :: 4',
     'Framework :: Wagtail',
-    'Framework :: Wagtail :: 4',
+    'Framework :: Wagtail :: 5',
 ]
 packages = [
     { include = "wagtail_airtable" },
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.7,<4.0"
-wagtail = ">=4.1"
+python = ">=3.8,<3.13"
```

Comment:
question: why not leave simply as `>=3.8`?

Reply:
No reason not to, updated in https://github.com/wagtail/wagtail-airtable/pull/56/commits/850822907ad31e5f78a82ab83c4c3fc6653e9338